pkg/lockfile/parse-apk-installed.go

```diff
@@ -0,0 +1,56 @@
+package lockfile
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+const AlpineEcosystem Ecosystem = "Alpine"
+
+func ParseApkInstalled(pathToLockfile string) ([]PackageDetails, error) {
+	var packages []PackageDetails
+
+	file, err := os.Open(pathToLockfile)
+	if err != nil {
+		return packages, fmt.Errorf("could not open %s: %w", pathToLockfile, err)
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+
+	var curPkg PackageDetails = PackageDetails{}
+
+	for scanner.Scan() {
+		line := scanner.Text()
+		if line == "" {
+			// First line is empty or multiple empty lines, no current package values
+			if (PackageDetails{}) == curPkg {
+				continue
+			}
+			// Empty line follows a package info block. Append package before going to next one
+			packages = append(packages, curPkg)
+			curPkg = PackageDetails{}
+			continue
+		}
+		// File SPECS: https://wiki.alpinelinux.org/wiki/Apk_spec
+		if strings.HasPrefix(line, "P:") {
+			curPkg.Name = strings.Split(line, "P:")[1]
+		} else if strings.HasPrefix(line, "V:") {
+			curPkg.Version = strings.Split(line, "V:")[1]
```

Comment:
I think it would be better to use `TrimPrefix` here since that is exactly what we're doing:

```suggestion
			curPkg.Version = strings.TrimPrefix(line, "V:")
```